app/services/airtable_placeholder_service.rb

```diff
@@ -90,8 +90,8 @@ def call(table_name)
             "In an effort to reduce spam and help maintainers,
             we’ve introduced a one-week review period for all pull requests.
             Once you have submitted four eligible PRs
-            (ready-to-review, not drafts), the seven-day review window begins.
-            This period gives maintainers time to
+            (ready-to-review, not drafts), the fourteen-day
+            review window begins. This period gives maintainers time to
             identify and label any spammy PRs as **invalid**. If your PRs are
             not marked invalid during that window,
             they will allow you to complete the Hacktoberfest challenge.
```

Comment:
It would be better to explicitly spell this out here

> If a maintainer reports your pull request as spam or behavior not in line with the project’s code of conduct, project’s code of conduct, you will be ineligible to participate. This year, the first 70,000 participants who	        you will be ineligible to participate